scripts/serialize_new_errata_rec.py

```diff
@@ -20,9 +20,12 @@
 from alws.models import NewErrataRecord, NewErrataPackage, Platform
 
 
-def serialize_model(instance, seen=None, exclude_models=None):
+def serialize_model(
+    instance, seen=None, exclude_models=None, exclude_columns=None
+):
     seen = set() if seen is None else seen
     exclude_models = [] if exclude_models is None else exclude_models
+    exclude_columns = [] if exclude_columns is None else exclude_columns
```

Comment:
I'm curious, is there a reason to don't just set the default values in the function definition? This is:
```
def serialize_model(
    instance, seen=set(), exclude_models=[], exclude_columns=[]
):
```

Reply:
Technically, in this case it's possible to make them default values since we only read these arguments.
But I think it's safer to keep existing implementation because of how python handles mutable default values.

Reply:
Ok, I was asking because we don't modify/write the lists or return any of these values. In that case I'd provide the defaults, but I leave it up to you. Thanks for the clarification

Reply:
Agree but from my experience it's too easy to introduce some changes that modify such argument and forget to move mutable part out from default value.